kayenta-core/src/main/java/com/netflix/kayenta/metrics/MetricsRetryConfigurationProperties.java

```diff
@@ -0,0 +1,17 @@
+package com.netflix.kayenta.metrics;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.http.HttpStatus;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Data
+@ConfigurationProperties("kayenta.retry")
+public class MetricsRetryConfigurationProperties {
+
+    private List<HttpStatus.Series> series = Arrays.asList(HttpStatus.Series.SERVER_ERROR);
```

Comment:
Is it worth adding 408s and 429s to this list and make people opt out of those codes rather than opt in, to maintain backwards compatibility.

Reply:
Ok, I will also add http statuses list that can be retried